resources/assets/js/pages/deploy_status.js

```diff
@@ -0,0 +1,8 @@
+var pathArray = window.location.pathname.split('/');
+var socket = io('http://'+ window.location.host +':3000');
+
+socket.on('output-channel:' + pathArray[2], function(data){
+    document.getElementById('output').innerHTML += data.output.replace(/(?:\r\n|\r|\n)/g, '<br/>');
```

Comment:
the element itself doesn't change, move this `getElementById` to variables and reuse it.